java/flight/flight-sql-jdbc-core/src/main/java/org/apache/arrow/driver/jdbc/accessor/impl/calendar/ArrowFlightJdbcTimeStampVectorAccessor.java

```diff
@@ -177,7 +174,7 @@ protected static TimeZone getTimeZoneForVector(TimeStampVector vector) {
 
     String timezoneName = arrowType.getTimezone();
     if (timezoneName == null) {
-      return TimeZone.getTimeZone("UTC");
+      return null;
```

Comment:
OK. This change seems to be the right one: an Arrow timestamp has no defined timezone if the timezone name is not set.

Reply:
+1